app/views/passengers/index.haml

```diff
@@ -15,7 +15,7 @@
     = check_box_tag :show_inactive, true, !@filters.include?('active')
   = submit_tag 'Show Passengers'
   = submit_tag 'Print These Filters', name: "print"
-  .checkbox-inline.key_position
+  .key_position
```

Comment:
Minor nit-pick: I'm not sure why this `div` is necessary. The `table` itself can be positioned just as easily.